### PR TITLE
From PascalCase to camelCase

### PR DESCRIPTION
**Pull request by StephenAraka**
### what does this do?
1. rename redux files that were in PascalCase to camelCase 

### BEFORE
**Actions**             |  **Reducers**
:-------------------------:|:-------------------------:
action files - BEFORE  |  reducer files - BEFORE

### AFTER
**Actions**             |  **Reducers**
:-------------------------:|:-------------------------:
action files - AFTER |  reducer files - AFTER

2. In the root reducer, rename imported functions that were in PascalCase to camelCase

**BEFORE**             |  **AFTER**
:-------------------------:|:-------------------------:
root reducer - BEFORE  |  root reducer - AFTER

### Some more context
I did this when the cluster was off... There were no compilation errors, however, I have not yet tested the changed namings.
After testing everything, I will label the PR `ready for review`.

### UPDATE
PR has been tested and everything is working okay.

### CODE REVIEW

File: src/components/ClusterPage/index.jsx [hunk 1]
```diff
@@ -198,14 +198,14 @@ ClusterPage.propTypes = {
   message: PropTypes.string.isRequired
 };
 
-const mapStateToProps = ({ user, AddClusterReducer }) => {
+const mapStateToProps = ({ user, addCluster }) => {
```
Comment (Mubangizi) [inline src/components/ClusterPage/index.jsx:201]: @StephenAraka please explain this to me, from the code, it looks like `addCluster` is an action, and you are getting `reducer` variables from it? Please help me understand it

Comment (StephenAraka) [reply src/components/ClusterPage/index.jsx:201]: @Mubangizi  there's addCluster the reducer and addCluster the action...
In that case, mapStateToProps refers to addCluster the reducer.

I renamed the reducer function.

Comment (Mubangizi) [reply src/components/ClusterPage/index.jsx:201]: No please append `Reducer` to it, for uniformity. like all the other `reducerNames`

Comment (StephenAraka) [reply src/components/ClusterPage/index.jsx]: done

**Pull request merged**